### PR TITLE
Google plus button now loads after load event

### PR DESCRIPTION
**Pull request by stephenkerrhs**
Update to the Google plus social button.

The button will now load once the "onload" event has been fired.

I have also remove some unnecessary code.

*_I should also mention that the button pulls in a Google font, Roboto_

### CODE REVIEW

File: lib/social_buttons/view_helpers/google_plus.rb [hunk 1]
```diff
@@ -8,28 +8,19 @@ module GooglePlus
...
 
       params = clazz.options_to_data_params(clazz.default_options.merge(options))
       params.merge!(class: CLASS)
 
-      scripter_class = use_script == :async ? clazz::AsyncScripter : clazz::Scripter
-
       html = "".html_safe
       html << content_tag(:div, nil, params)
-      html << scripter_class.new(self).script(locale) if use_script
+      html << clazz::Scripter.new(self).script(locale)
```
Comment (pusewicz) [inline lib/social_buttons/view_helpers/google_plus.rb]: I think `klass` is a more widely used variable name.

**Comment by pusewicz** [comment]
:+1:

File: lib/social_buttons/view_helpers/google_plus.rb (hunk 1 above)
Comment (stephenkerrhs) [inline lib/social_buttons/view_helpers/google_plus.rb]: I'll change that now.

**Comment by dobrien501** [comment]
:+1:

**Comment by zeto** [comment]
:+1:

**Pull request merged**